packages/lime-system/files/usr/lib/lua/lime/get_ifname.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ -n "$1" ] then
+    . /usr/share/libubox/jshn.sh
+
+    json_load "$(cat /etc/board.json)"
+    json_select network
+    if json_get_type Type "$1" && [ "$Type" == object ]; then
```

Comment:
`[ "$Type" == "object" ]` is a bit more readable (mind the quotes around the `object` string, not needed strictly)